nanogpt/export_nanogpt.py

```diff
@@ -24,11 +25,29 @@
 )
 
 
+class NanoGPT(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.model = GPT.from_pretrained('gpt2') # use gpt2 weight as pretrained weight
+
+    def forward(self, idx, max_new_tokens):
+        for _ in range(20):
```

Comment:
Please comment on what 20 is and how we chose that number; better to move it to a top-level CONSTANT with a comment on it.